Game/Source/GDKTestGyms/Private/TestGymsReplicationGraph.cpp

```diff
@@ -855,12 +855,12 @@ void UTestGymsReplicationGraphNode_GlobalViewTarget::GatherActorListsForConnecti
 
 	for (const FNetViewer& CurViewer : Params.Viewers)
 	{
-		if (CurViewer.InViewer)
+		if (CurViewer.InViewer != nullptr)
```

Comment:
```
if (IsValid(CurViewer.InViewer))
```

Reply:
Have seen that ConditionalAdd actually does validation checks internally, so have removed this check entirely.